src/components/WishListPage/WishListPage.js

```diff
@@ -47,7 +47,7 @@ class WishList extends Component {
             {/* Mapping through our item reducer to display remaining items */}
             {this.props.state.list.map((item) => {
               if (item.priority === false) {
-                return <ul key={item.id}>{item.item}</ul>;
```

Comment:
I think there's way we could denote that text as clickable without having to have the item and a url displaying on the DOM :)